src/core/contracts/registry.test.ts

```diff
@@ -29,7 +29,6 @@ import {
 import { generateHexString } from "@dsnp/test-generators";
```

Comment:
Why all these changes to the registry test? Well a flaky test finally started failing consistently. The blockNumber wasn't always returning the correct value from the hardhat ganache with mineBlocks.